CMakeLists.txt

```diff
@@ -22,15 +22,20 @@ cmake_minimum_required(VERSION 3.10...3.17.2)
 
 project(OpenSSL VERSION "1.0.0" LANGUAGES CXX DESCRIPTION "CMake to compile OpenSSL" HOMEPAGE_URL "https://github.com/flagarde/openssl-cmake")
 
-set(OPENSSL_TAG "1_1_1i" CACHE STRING "Version the OpenSSL to download")
+if(OPENSSL_VER)
+  string(REPLACE "." "_" OPENSSL_TAG ${OPENSSL_VER})
+else()
+   set(OPENSSL_TAG "1_1_1j" CACHE STRING "Version the OpenSSL to download")
+endif()
+
```

Comment:
Why not directly : 
```cmake
set(OPENSSL_VER "1_1_1j")
string(REPLACE "." "_" OPENSSL_TAG ${OPENSSL_VER})
```

Reply:
It should let user specify a different value other than latest

Reply:
ok, so maybe 
```cmake
if(NOT DEFINED OPENSSL_VER)
  set(OPENSSL_VER "1_1_1j")
endif()
string(REPLACE "." "_" OPENSSL_TAG ${OPENSSL_VER})
```